File: src/2643.row-with-maximum-ones.2737/solution.ts
function rowAndMaximumOnes (mat: number[][]): number[] {
    let index = -1;
    let count = -1;
    for (let i = 0; i < mat.length; i++) {
        let c = 0;
        for (let j = 0; j < mat[0].length; j++) {
            if (mat[i][j] === 1) {
                c++;
            }
        }
        if (c > count) {
            index = i;
            count = c;
        }
    }

    return [index, count, ];
}

File: src/2643.row-with-maximum-ones.2737/solution.java
class Solution {
    public int[] rowAndMaximumOnes(int[][] mat) {
        int row = 0;
        int cnt = 0;
        int M = mat.length;
        for(int i=0;i<M;i++){
            int c = 0;
            for(int n : mat[i]){
                c += n;
            }
            if(c>cnt){
                row = i;
                cnt = c;
            }
        }
        return new int[]{row,cnt};
    }
}